### PR TITLE
Milestone 4: Style for desktop version with media queries

### PR DESCRIPTION
**Pull request by cnualartc**
Used media queries to style the desktop version of the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,16 +12,27 @@
...
         <a href="">My Logo</a>
       </div>
       <nav class="menu">
+        <div class="desktop-menu-list-container">
+          <ul class="desktop-menu-list">
+            <li class="menu-desktop">Portfolio</li>
+            <li class="menu-desktop">About</li>
+            <li class="menu-desktop">Contact</li>
+          </ul>
+        </div>
```
Comment (leonardodiasb) [inline index.html]: - [x] Your navbar is very similar to the Figma template. But the background color is not filling the entire navbar, and there are places with the correct white background color and other places with a gray background color making it not look similar to the template and not professional. Kindly make the whole navbar background white.
 - [x] Also the nav items are correctly wrapped with the `li` tags but they are also links to the sections of the page, so an `a` tag inside the `li` would be appropriate for this case. Kindly fix that.
 - Your design:
 - 
image

File: style.css [hunk 2]
```diff
@@ -382,3 +412,300 @@ textarea {
...
+    justify-content: start;
+    font-size: 12px;
+  }
+
+  .card-button {
+    width: 496px;
+    grid-area: button;
+    display: flex;
+    justify-content: start;
+  }
```
Comment (leonardodiasb) [inline style.css]: - [x] Your project cards are similar to the Figma template, but they are not responsive and on smaller screen sizes they are cut out off the screen. Please make them responsive so they can fit properly, maintaining the dimensions, on every screen size.
 - [x] There are some areas inside the project card with different background colors. Please make them consistent and use the white background color for all the cards to match the Figma template.
 - [x] The whole card has a cursor behavior as pointer, but when the user hovers over the button the cursor behavior is normal. This is inverted behavior. The expected behavior is to be pointer when the user hovers over the button and normal on the rest of the card so the user can know that that button is clickable. Kindly change that.
 - Your design:
 - 
image

File: style.css [hunk 3]
```diff
@@ -382,3 +412,300 @@ textarea {
...
+    justify-items: start;
+    align-items: start;
+    grid-template-areas:
+      "blank languages languages"
+      "title languages languages"
+      "paragraph languages languages"
+      "connect languages languages"
+      "icons languages languages"
+      "button languages languages";
+  }
```
Comment (leonardodiasb) [inline style.css]: - [x] The about section is great and very similar, but there are some points that makes different from the template. First the background color of the about section should be white, to differentiate from the previous projects section background color and there should be a border radius on the top right corner. Kindly change that.
 - [x] Also on smaller screen sizes, there is no padding on the sides of this section. Kindly add some padding to match the Template.
 - Your design:
 - 
image

 - Figma template:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @cnualartc, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of GitHub flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @cnualartc, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow
- Requested  changes have been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**